samples/react-timeline/src/webparts/timeline/loc/en-us.js

```diff
@@ -0,0 +1,13 @@
+define([], function() {
+  return {
+    "PropertyPaneDescription": "Description",
+    "BasicGroupName": "Group Name",
+    "DescriptionFieldLabel": "WebPart Title",
+    "ListNameFieldLabel":"List Name",
+    "LayoutFieldLabel":"Layout Type",
+    "ShowImageFieldLabel":"Show Image",
+    "ShowDescriptionFieldLabel": "Show Description",
+    "DateFormatFieldLabel": "Date Format",
+    "SortOrderFieldLabel": "Sort Direction"
+  }
+});
```

Comment:
You have a localization file, I'm curious why you don't localize the other literals in your solution?